.appveyor.yml

```diff
@@ -0,0 +1,23 @@
+# Operating system (build VM template)
+os: Visual Studio 2015
```

Comment:
my documentation is for VS2017 if this is not available you need to change the cmake line to `-T v140,host=x64` or you skip the -T parameter